pycytominer/cyto_utils/load.py

```diff
@@ -69,13 +77,66 @@ def infer_delim(file: str):
     return dialect.delimiter
 
 
-def load_profiles(profiles):
+def is_anndata(
+    path: Union[str, pathlib.Path, ad._core.anndata.AnnData],
+) -> tuple[bool, Optional[str]]:
+    """Return True if ``path`` contains an AnnData dataset (H5AD or Zarr).
+
+    This function prefers using the AnnData readers directly:
+    - in-memory AnnData objects are recognized directly.
+    - H5AD files are opened in backed mode to avoid loading data into memory.
```

Comment:
what is "backed mode"?

Reply:
Thanks @flying-sheep and @ilan-gold for the comments here! I'll add some notes to the code comments in reference to this. For now I'll plan to keep using backed mode until `read_lazily` matures from experimental status.